source/change-streams/change-streams.rst

```diff
@@ -481,10 +481,19 @@ Once a ``ChangeStream`` has encountered a resumable error, it MUST attempt to re
 - Connect to selected server.
 - If there is a cached ``resumeToken``:
 
-  - The driver MUST set ``resumeAfter`` to the cached ``resumeToken``.
-  - The driver MUST NOT set ``startAfter``. If ``startAfter`` was in the original aggregation command, the driver MUST remove it.
-  - The driver MUST NOT set ``startAtOperationTime``. If ``startAtOperationTime`` was in the original aggregation command, the driver MUST remove it.
-- If there is no cached ``resumeToken`` and the ``ChangeStream`` has a saved operation time (either from an originally specified ``startAtOperationTime`` or saved from the original aggregation) and the max wire version is >= ``7``:
+  - If the ``ChangeStream`` was started with ``startAfter`` and has yet to return a result document:
+
+    - The driver MUST set ``startAfter`` to the cached ``resumeToken``.
+    - The driver MUST NOT set ``resumeAfter``.
```

Comment:
I take it that there is no need for the "If ``resumeAfter`` was in the original aggregation command..." copypasta here because the original `aggregate` would have failed with a server error due to using both options at once and we would never get as far as needing to resume?

Reply:
That's right. Since an `aggregation` with both those options would fail anyway (with a non-resumable error), I didn't think it made sense to explicitly specify behavior for that case.